Makefile

```diff
@@ -289,7 +289,21 @@ start-jupyter-notebook: build-release-image-jupyter ## Start Jupyter Notebook en
 
 .PHONY: start-jupyter-notebook
 
-debug-jupyter-notebook: build-release-image-jupyter ## Debug jupyter notebook using the ostk-astro package built from current source code, must have run make start-development and ostk-build and ostk-install-python to use this
+debug-jupyter-notebook: build-development-image build-release-image-jupyter ## Debug jupyter notebook using the ostk-astro package built from current source code
+
+	@ echo "Building Python$(jupyter_python_version) packages..."
+	@ mkdir -p $(CURDIR)/build
+	@ mkdir -p $(CURDIR)/packages/python
+
+	docker run \
+		-it \
+		--rm \
+		--volume="$(CURDIR):/app:delegated" \
+		--workdir=/app/build \
+		$(docker_development_image_repository):$(docker_image_version) \
+		/bin/bash -c "cmake -DBUILD_UNIT_TESTS=OFF -DBUILD_BENCHMARK=OFF -DBUILD_PYTHON_BINDINGS=ON -DPYTHON_SEARCH_VERSIONS="$(jupyter_python_version)" .. \
+		&& $(MAKE) -j $(shell nproc) \
+		&& cp /app/build/bindings/python/dist/*.whl /app/packages/python"
```

Comment:
Honestly this is a bit "worse" for the dev experience because even if I have built the binary this will rebuild it. SO I have to wait for it to build which takes a while.
If you want to include this I would make it a flag as this also prevents you from using a custom binary. For example if I wanted to build it in debug mode or compile it with very specific flags this will not allow me to do so.

Reply:
Yup good point, changed it, let me know what you think.